docs/event.md

```diff
@@ -48,18 +60,18 @@ Used to either create or send mock events for use with local webhooks testing.
 | `--to-user`         | `-t`      | Denotes the receiver's TUID of the event, usually the broadcaster.                                                              | `-t 44635596`                             | N               |
 | `--from-user`       | `-f`      | Denotes the sender's TUID of the event, for example the user that follows another user or the subscriber to a broadcaster.      | `-f 44635596`                             | N               |
 | `--gift-user`       | `-g`      | Used only for subcription-based events, denotes the gifting user ID                                                             | `-g 44635596`                             | N               |
 | `--secret`          | `-s`      | Webhook secret. If defined, signs all forwarded events with the SHA256 HMAC.                                                    | `-s testsecret`                           | N               |
-| `--count`           | `-c`      | Count of events to fire. This can be used to simulate an influx of subscriptions.                                               | `-c 100`                                  | N               |
+| `--count`           | `-c`      | Count of events to fire. This can be used to simulate an influx of events.                                                      | `-c 100`                                  | N               |
 | `--anonymous`       | `-a`      | If the event is anonymous. Only applies to `gift` and `cheer` events.                                                           | `-a`                                      | N               |
 | `--status`          | `-S`      | Status of the event object, currently applies to channel points redemptions.                                                    | `-S fulfilled`                            | N               |
 | `--item-id`         | `-i`      | Manually set the ID of the event payload item (for example the reward ID in redemption events or game in stream events).        | `-i 032e4a6c-4aef-11eb-a9f5-1f703d1f0b92` | N               |
 | `--item-name`       | `-n`      | Manually set the name of the event payload item (for example the reward ID in redemption events or game name in stream events). | `-n "Science & Technology"`               | N               |
 | `--cost`            | `-C`      | Amount of bits or channel points redeemed/used in the event.                                                                    | `-C 250`                                  | N               |
-| `--description`     | `-d`      | Title the stream should be updated/started with.                                                                                | `-d Awesome new title!`                   | N               |
+| `--description`     | `-d`      | Title the stream should be updated/started with. Additionally used as the category ID for Drops events.                         | `-d Awesome new title!`                   | N               |
```

Comment:
I don't see how the category ID is used in this command. The description is used to adjust the category ID for Drops-related events?

Reply:
Correct. Unfortunately there's not a great identifier elsewhere to use since ItemID is used for the Drop itself.

Reply:
I don't quite agree with overloading the description parameter here when the CLI already generates a random numerical ID for the category ID if this param is missing. It's not breaking anything, but seems unnecessary when instead there could just be a custom param for category_name with a category_id autogenerated based on the string. That's more of a feature request at this point, so it's not something I'd block the release on.

Reply:
Totally fair, and I can remove the overload. We're getting into the weeds of needing to create a new flag (which is fine) but all the variations of "c" are taken now, so have to make a new one (possibly "g" for game?). Let me make that edit.

Reply:
Added a new flag to address above. :)